atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java

```diff
@@ -402,4 +408,30 @@ public void addCleanupTask(String rawTableName, Supplier<OnCleanupTask> task) {
     public void ignoreTableNameLengthChecks() {
         ignoreTableNameLengthChecks = true;
     }
+
+    public SchemaMetadata getSchemaMetadata() {
+        // TODO (jkong): Memoize?
```

Comment:
Probably not, since the definitions can change?

Reply:
I guess the intention was to have a flag that indicates whether the computed version is stale and/or for mutations like `addTableDefinition` to null out the computed reference. We can revisit this if there's a need for perf, though in the latest version I've removed this comment.